site/app/models/User.php

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Set:
https://github.com/Submitty/Submitty/blob/5cf9cdc1a7f31db964a250ec737af4db3c886e92/site/app/models/User.php#L194-L196

to be $this->time_zone = $details['time_zone'] ?? 'NOT_SET/NOT_SET';

Reply:
Changed.